tei/poliziano-orfeo.xml

```diff
@@ -653,7 +667,8 @@
           <l>Or vadi e biasimi la teda legittima!</l>
           <l>Euoè Bacco! accepta questa vittima!</l>
         </lg>
-        <sp who="#el_coro_delle_baccante">
+        </sp>
+        <sp who="#coro_delle_baccanti">
```

Comment:
We should probably stick to the text "El coro delle BACCANTE", also for the ID, seems to be the correct spelling in the print source:

![baccante](https://user-images.githubusercontent.com/6539515/181054944-41a6491b-c0c1-497b-bb18-871ad31cadca.png)

Reply:
Thanks for pointing out! Since the IDs from the particDesc are the ones used as labels for the network visualisation, I thought using the modernised Italian version for them (while obviously not modifying the in-text `<speaker>` elements) would have improved readability. I'll take note of this practice!

Reply:
thanks, i usually sticked to historic spelling also for the IDs in all the corpora that i built/adapted… i think that modernising the spelling of names/speakers would add all kinds of new problems, especially if you wanna be consistent…

Reply:
fixed!